### DISCUSSION TITLE
language code

### DISCUSSION THREAD

**Discussion by dalu** [dr5hn/countries-states-cities-database#414] [Ideas]
hello
why you don't use 3 letters for language code ? isn't it standard ?
2 letters is old school ?
we use 3 letters
thanks

**Comment by dr5hn** [comment]
Hi

I didn't get your point, can you please explain me via screenshot what are
you referring to?

Thanks

**Comment by dalu** [comment]
<img width="425" alt="image" src="https://user-images.githubusercontent.com/46044550/158378147-b8412413-90c6-4c1e-92d3-9ae9b72731be.png">

**Comment by dalu** [comment]
iso -639 is 3 letters 
<img width="297" alt="image" src="https://user-images.githubusercontent.com/46044550/158379153-a06c228d-fc85-4eea-956b-926e14d6b28a.png">

**Comment by dalu** [comment]
what do you think ? i trying to find ways to store translations

**Comment by dr5hn** [comment]
Hi

It's a good suggestion, however not sure how to organize it in the current
structure. Need more time to think about it.

On a side note, actioning could take more than a month.

Thanks & Regards,
*Darshan Gada*
*9029208698* <+919029208698>
